src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs

```diff
@@ -207,9 +262,7 @@ public static void EndInvoke(long asyncHandle, bool succeeded, JSAsyncCallResult
         [JSInvokable(nameof(DotNetDispatcher) + "." + nameof(ReleaseDotNetObject))]
         public static void ReleaseDotNetObject(long dotNetObjectId)
```

Comment:
Should this whole `[JSInvokable]` method still exist? It looks like elsewhere, you're removing support for issuing a disposal call from the JS side.

Reply:
I left the support for disposing it from client, but removed support for untracking the object by calling `IJSRuntime.UntrackObjectRef`. In .NET, you can untrack it by disposing the reference. Are there any scenarios I'm missing where `UntrackObjectRef` is useful?

Reply:
I still haven't yet understood exactly how one now disposes it from the client. What specific JS code achieves this? When I make sense of the change enough to understand that, I can probably answer your question :)

Reply:
Update: Now I understand it. I agree with your reasoning - we don't need `UntrackObjectRef`.